src/org.hdfgroup.hdfview/hdf/view/ViewProperties.java

```diff
@@ -93,6 +93,7 @@
     public static final String DEFAULT_SCALAR_DATASET_TABLEVIEW_NAME = "hdf.view.TableView.DefaultScalarDSTableView";
     public static final String DEFAULT_SCALAR_ATTRIBUTE_TABLEVIEW_NAME = "hdf.view.TableView.DefaultScalarAttributeTableView";
     public static final String DEFAULT_COMPOUND_DATASET_TABLEVIEW_NAME = "hdf.view.TableView.DefaultCompoundDSTableView";
+    public static final String DEFAULT_COMPOUND_ATTRIBUTE_TABLEVIEW_NAME = "hdf.view.TableView.DefaultCompoundAttributeTableView";
```

Comment:
Probably need to delete this key and the one above for ScalarAttributeTableView and ensure they get mapped appropriately to ScalarDSTableView and CompoundDSTableView instead.

Reply:
Right, I need to do extra work in this file because of other changes not concerning this issue.

Reply:
Taken care of in a secondary commit.